easybuild/easyconfigs/j/jbigkit/jbigkit-2.1_fix-libpath.patch

```diff
@@ -0,0 +1,26 @@
+build process does not find its own libraries
+
```

Comment:
Please add some Author information here. And don't forget to update the checksum too